sdk/python/pyproject.toml

```diff
@@ -0,0 +1,51 @@
+[project]
+name = "datastar-py"
+description = "Helper functions and classes for the Datastar library (https://data-star.dev/)"
+readme = "README.md"
+authors = [
+    { name = "Felix Ingram", email = "f.ingram@gmail.com" }
+]
+requires-python = ">=3.9"
+dependencies = []
+dynamic=["version"]
+
```

Comment:
...and a new section to show URLs in pypi page.

```toml
[project.urls]
Documentation = "https://github.com/starfederation/datastar/blob/develop/sdk/python/README.md"
GitHub = "https://github.com/starfederation/datastar"
```

Reply:
Good suggestion.